### ISSUE TITLE
NTR: 27-hydroxycholesterol high in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8387]
New term request 27-hydroxycholesterol high in blood (elthat_150815135732):   UMLS: 4231023

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000402
- label: Elevated circulating 27-hydroxycholesterol concentration
- parents: [Abnormal circulating cholesterol concentration (HP:0003107)](https://hpo.jax.org/app/browse/term/HP:0003107)
- pmid: PMID:24288332
- def: The concentration of 27-hydroxycholesterol in the blood circulation is above the upper limit of normal.
- comment: 27-hydroxycholesterol is a primary metabolite of cholesterol.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8387
- synonyms:
